guides/issuing-assets.md

```diff
@@ -39,7 +39,7 @@ In the simplest situations, you can issue assets from your everyday Stellar acco
 
 ### Requiring or Revoking Authorization
 
-Accounts have [several flags](concepts/accounts.md#flags) related to issuing assets. You’ll almost always want to make sure your issuing account has the [`AUTHORIZATION REVOCABLE` flag set](concepts/assets.md#revoking-access). This allows you to freeze your assets in another account.
+Accounts have [several flags](concepts/accounts.md#flags) related to issuing assets. You might want to make sure your issuing account has the [`AUTHORIZATION REVOCABLE` flag set](concepts/assets.md#revoking-access). This allows you to freeze your assets in another account.
```

Comment:
So @bartekn described this to me before as being something that asset issues should be using the vast majority of the time—or at least that we should use stronger language encouraging people to use it (I think this was while drafting the security guide, where it now says “If you issue your own assets, you should usually ensure that they can be revoked”).

If “almost always” is too strong, is “might” strong enough?